builder/builder-config/src/main/java/io/helidon/builder/config/spi/GeneratedConfigBeanBuilderBase.java

```diff
@@ -65,6 +65,17 @@ public void __config(Config cfg) {
         this.cfg = Objects.requireNonNull(cfg);
     }
 
+    /**
+     * Update status of this builder from configuration, overriding existing values if config defines them.
+     *
+     * @param cfg configuration to update this builder from
```

Comment:
bad english. change to "the configuration source to use in order to update this builder"

Reply:
just removed the word source from your text, because that is misleading (it has a very specific meaning in configuration module).